doc/source/myrocks/limitations.rst

```diff
@@ -23,7 +23,8 @@ The MyRocks storage engine lacks the following features compared to InnoDB:
 You should also consider the following:
 
 * :file:`*_bin` (e.g. ``latin1_bin``) or binary collation should be used
-  on ``CHAR`` and ``VARCHAR`` indexed columns.
+  on ``CHAR`` and ``VARCHAR`` indexed columns. The following binary collations
+  are supported: ``binary``, ``latin1_bin``, and ``utf8_bin``.
```

Comment:
This is true for PS-5.7.
From PS-8.0.23 all collations are now supported for `CHAR` and `VARCHAR` indexed columns.
Implemented with https://jira.percona.com/browse/PS-7310

Reply:
Removed the binary columns sentence.